docs/docs/modules/data_connection/document_loaders/sqlalchemy.mdx

```diff
@@ -0,0 +1,199 @@
+# SQLAlchemy
+
```

Comment:
Most document loaders are only documented in `integrations`.
Shouldn't this be the case for this one too ?

Reply:
87cd992b916 absorbs the relevant documentation sections into the notebook, like you suggested.